app/controllers/widget_tests_controller.rb

```diff
@@ -1,4 +1,4 @@
-class WidgetTestsController < ActionController::Base
+class WidgetTestsController < ApplicationController
```

Comment:
WidgetTests do not need to inherit ApplicationController for the reasons mentioned above